src/templates/ProjectLandingPage/index.js

```diff
@@ -66,9 +59,14 @@ const DetailsPage = ({ pageContext: { project } }) => {
         {project.route ? (
           <meta
             property="og:url"
-            content={`https://labs.moduscreate.com/${project.route}`}
+            content={`https://labs.moduscreate.com/${project.route}/`}
```

Comment:
Can we use something like `location.origin` instead of hardcoding the URL?

Reply:
Yes, I plan to create a new PR to remove hard coded string.

Reply:
Come to think of it, `location.origin` will not work. We'll need to use gatsby config + ENV vars to set the URL. I'll create an issue for that task